tracing-subscriber/CHANGELOG.md

```diff
@@ -1,3 +1,29 @@
+# 0.3.18 (November 6, 2023)
+
+This release of `tracing-subscriber` adds support for `NO_COLOR` in
+`fmt::Layer` and reintroduces support for the `chrono` crate.
+It also introduces several minor API improvements.
+
+
+### Added
+
+- **chrono**: Add `chrono` implementations of `FormatTime` ([#2690])
+- **subscriber**: support `NO_COLOR` in `fmt::Layer` ([#2647])
+
+### Changed
+
+- **fmt**: make `format::Writer::new()` public ([#2680])
+- **subscriber**: bump version of `tracing-log` to 0.2 ([#2772])
```

Comment:
i think this should probably be:
```suggestion
- **log**: bump version of `tracing-log` to 0.2 ([#2772])
```
because normally, we use the _feature flag_ for these

Reply:
whoops! wasn't aware, sorry!